eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -48,87 +48,69 @@ param(
   [Parameter(Mandatory = $false)]
   [string]$PRBody = $PRTitle,
 
-  [Parameter(Mandatory = $false)]
-  [string]$PRLabels
-)
-
-$headers = @{
-  Authorization = "bearer $AuthToken"
-}
+  [string]$PRLabels,
 
-$query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
+  [string]$UserReviewers,
 
-function AddLabels([int] $prNumber, [string] $prLabelString, [array] $existingLabels)
-{
-  # Adding labels to the pr.
-  if (-not $prLabelString) {
-    Write-Verbose "There are no labels added to the PR."
-    return
-  }
+  [string]$TeamReviewers,
 
-  # Parse the labels from string to array
-  $prLabelArray = @($prLabelString.Split(",") | % { $_.Trim() } | ? { return $_ })
-  foreach ($label in $existingLabels) {
-    if ($prLabelArray -contains $label.name) {
-      continue
-    }
-    $prLabelArray += $label.name
-  }
-  $prLabelUri = "https://api.github.com/repos/$RepoOwner/$RepoName/issues/$prNumber"
-  $labelRequestData = @{
-    labels = $prLabelArray
-  }
-  try {
-    $resp = Invoke-RestMethod -Method PATCH -Headers $headers $prLabelUri -Body ($labelRequestData | ConvertTo-Json)
-  }
-  catch {
-    Write-Error "Invoke-RestMethod $prLabelUri failed with exception:`n$_"
-  }
+  [string]$Assignees
+)
 
-  $resp | Write-Verbose
-  Write-Host -f green "Label(s) [$prLabelArray] added to pull request: https://github.com/$RepoOwner/$RepoName/pull/$prNumber"
-}
+. "${PSScriptRoot}\common.ps1"
 
 try {
-  $resp = Invoke-RestMethod -Headers $headers "https://api.github.com/repos/$RepoOwner/$RepoName/pulls?$query"
+  $resp = List-GithubPullRequests -RepoOwner $RepoOwner -RepoName $RepoName `
+  -Head "${PROwner}:${PRBranch}" -Base $BaseBranch
 }
 catch { 
-  Write-Error "Invoke-RestMethod [https://api.github.com/repos/$RepoOwner/$RepoName/pulls?$query] failed with exception:`n$_"
+  LogError "List-GithubPullRequests failed with exception:`n$_"
```

Comment:
We should verify that the exception message contains the url for the exception as that is going to be interesting if there is an error.

Reply:
Am not sure that this is the case. How about if I catch the exceptions in the `Invoke-GitHubAPI` function, log the uri and parameters then rethrow the exception.